components/event-bus/internal/knative/util/kn-eventing.go

```diff
@@ -114,7 +114,7 @@ func GetKnativeLib() (*KnativeLib, error) {
 func (k *KnativeLib) GetChannel(name string, namespace string) (*evapisv1alpha1.Channel, error) {
 	channel, err := k.evClient.Channels(namespace).Get(name, metav1.GetOptions{})
 	if err != nil {
-		//log.Printf("ERROR: GetChannel(): getting channel: %v", err)
+		log.Printf("ERROR: GetChannel(): getting channel: %v", err)
```

Comment:
It's a good practice to log in the caller, not in the library. If the intention was to add context, how about wrapping your error inside `fmt.Errorf` instead?